server/routes/route-helpers.ts/users-helpers.ts

```diff
@@ -0,0 +1,62 @@
+import { client } from "../../src/index";
+
+export const createUser = async (req: any, res: any) => {
+  await client
+    .query(
+      `
+        INSERT INTO users ("userName", "firebaseAuthId", "profileImgUrl")
+        VALUES ('${req.body.userName}', '${req.body.firebaseAuthId}', '${req.body.profileImgUrl}');
+      `
+    )
+    .then(() => res.send("you have successfully created a user"))
+    .catch((e: any) => res.json({ error: e }));
+};
+
+export const deleteUserById = async (req: any, res: any) => {
+  await client
+    .query(
+      `
+			DELETE FROM users WHERE "userId" = '${req.params.userId}';
+		`
+    )
+    .then(() =>
+      res.send("you have successfully deleted a user uid: " + req.params.userId)
+    )
+    .catch((e: any) => res.json({ error: e }));
```

Comment:
same here, `res.status(404).json({error: e})`

read more about what 404 error is [here](https://www.dotcom-monitor.com/blog/2022/03/03/the-10-most-common-http-status-codes/#:~:text=Status%20Code%20404%20%E2%80%93%20The%20most,due%20to%20improper%20URL%20redirection.)

also other places also need status codes too. I won't put a comment in all of them.